Sources/Fuzzilli/Core/FuzzEngine.swift

```diff
@@ -27,6 +27,7 @@ extension FuzzEngine {
 
         switch execution.outcome {
             case .crashed(let termsig):
+                program.comments.add("Program crash with options: \(fuzzer.runner.processArguments)", at: .footer)
```

Comment:
The minimizer was changed such that the comments are kept during minimization

Reply:
And I think it actually makes sense in general (even when not doing argument randomization) to include the process arguments in the crashing sample as comment, since they are always needed for reproing crashes